server/athenian/api/internal/miners/github/deployment.py

```diff
@@ -503,7 +509,10 @@ async def _postprocess_deployed_releases(
         return pd.DataFrame()
     assert isinstance(releases.index, pd.Int64Index)
     releases.sort_values(
-        Release.published_at.name, ascending=False, ignore_index=True, inplace=True,
+        Release.published_at.name,
+        ascending=False,
+        ignore_index=True,
+        inplace=True,
```

Comment:
This shouldn't happen :thinking:

Reply:
maybe I did the wrong add-trailing + chorny dance while rebasing from pre-black to post-black, let's see if it happen again in the normal workflow

Reply:
reverted